packages/ddp-client/common/livedata_connection.js

```
@@ -1416,13 +1416,23 @@ export class Connection {
     const oldOutstandingMethodBlocks = self._outstandingMethodBlocks;
     self._outstandingMethodBlocks = [];
 
-    self.onReconnect && self.onReconnect();
-    DDP._reconnectHook.each((callback) => {
-      callback(self);
+    const results = [];
+    if (self.onReconnect) {
+      results.push(self.onReconnect());
+    }
+    DDP._reconnectHook.forEach((callback) => {
```

ESLint

[error] Replace `(callback)` with `callback`

(prettier/prettier)